core/src/Cache/Keys/MsalAccountCacheKey.cs

```diff
@@ -39,9 +39,9 @@ internal class MsalAccountCacheKey
         private readonly string _environment;
         private readonly string _homeAccountId;
         private readonly string _tenantId;
-        private readonly string _localAccountId;
+        private readonly string _username;
```

Comment:
>_username [](start = 32, length = 9)

why the switch to _username?

Reply:
this field is used only for persistence on iOS - as value for "Generic", iOS team requested this change   
 to optimize cache lookup, from .Net side we need to change it to be compatible with iOS Msal